File: Java/Builder/GlassBuilder.java

public class GlassBuilder implements Builder{
	
	private ComplexObject complexObject;
	
	public GlassBuilder() {
		this.complexObject = new ComplexObject();
	}

	@Override
	public void buildNumber() 
	{
		this.complexObject.setNumberOfStuff(20);
	}

	@Override
	public void buildValue() {
		this.complexObject.setValue(200);
	}

	@Override
	public void buildDefinition() {
		this.complexObject.setDefinition("This a builder for glass that is way over priced");
	}

	@Override
	public ComplexObject getObject() {
		return this.complexObject;
	}

}

File: Swift/Patterns/Bridge/Bridge/GlassBuilder.swift
//
//  GlassBuilder.swift
//  Bridge
//
//  Created by colan biemer on 4/4/15.
//  Copyright (c) 2015 colan biemer. All rights reserved.
//

import Foundation

class GlassBuilder: Builder
{
	var obj: ComplexObject
	
	init()
	{
		obj = ComplexObject()
	}
	
	func buildNumber()
	{
		obj.setNum(200)
	}
	func buildValue()
	{
		obj.setVal(200)
	}
	func buildDefinition()
	{
		obj.setDef("This is glass")
	}
	func getObj() -> ComplexObject
	{
		return obj
	}
}